en/docs/learn/enterprise-integration-patterns/message-construction/event-message.md

```diff
@@ -13,89 +13,167 @@ The Event Message EIP is used for reliable, asynchronous event notification betw
+The example scenario depicts an inventory for stocks and demonstrates how the EIP distributes a message among several subscribers. It includes multiple instances of the subscribers for the event. When a message is sent to the WSO2 MI, it is transmitted to these subscriber instances, each of which acts as a subscriber through the ActiveMQ topic.
 
-The diagram below depicts how to simulate the example scenario using the ESB profile.
+The diagram below depicts how to simulate the example scenario using the WSO2 MI.
 
 ![Event message]({{base_path}}/assets/img/learn/enterprise-integration-patterns/message-construction/event-message.png)
 
 Before digging into implementation details, let's take a look at the relationship between the example scenario and the Event Message EIP by comparing their core components.
 
-| Event Message EIP (Figure 1) | Event Message Example Scenario (Figure 2) |
+| Event Message EIP            | Event Message Example Scenario            |
 |------------------------------|-------------------------------------------|
 | Subject                      | Stock Quote Client                        |
-| Event Message                | Event, Topic                              |
-| Observer                     | Stock Quote Service Instance              |
+| Event Message                | ActiveMQ topic                            |
+| Observer                     | EventObserver Proxies                     |
+
+## Synapse configuration of the artifacts
+
+=== "EventGeneratorProxy"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <proxy name="EventGeneratorProxy" transports="http" startOnLoad="true" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+       <target>
+          <inSequence>
+             <property name="OUT_ONLY" value="true" />
+             <property name="FORCE_SC_ACCEPTED" value="true" scope="axis2" />
+             <call>
+                   <endpoint>
+                      <address uri="jms:/SimpleStockQuoteService?transport.jms.ConnectionFactoryJNDIName=TopicConnectionFactory&amp;java.naming.factory.initial=org.apache.activemq.jndi.ActiveMQInitialContextFactory&amp;java.naming.provider.url=tcp://localhost:61616&amp;transport.jms.DestinationType=topic"/>
+                </endpoint>
+             </call>
+          </inSequence>
+       </target>
+    </proxy>
+    ```
+=== "EventObserver1"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <proxy name="EventObserver1" transports="jms" startOnLoad="true" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+       <target>
+           <inSequence>
+              <property name="OUT_ONLY" value="true"/>
```

Comment:
We don't need the `OUT_ONLY` property here as are not forwarding the message. Shall we test it and update this.